cabal-install/Distribution/Client/CmdInstall.hs

```diff
@@ -659,9 +660,10 @@ installExes verbosity baseCtx buildCtx platform compiler
       mkFinalExeName exe = prefix <> unUnqualComponentName exe <> suffix <.> exeExtension platform
       installdirUnknown =
         "installdir is not defined. Set it in your cabal config file "
-        ++ "or use --installdir=<path>"
+        ++ "or use --installdir=<path>. Using default installdir."
```

Comment:
It would be convenient if the default path was displayed here as well.

Reply:
Done. New format:

```
Warning: installdir is not defined. Set it in your cabal config file or use
--installdir=<path>. Using default installdir: "/home/mrenaud/.cabal/bin"
```